openquake/hazardlib/sourceconverter.py

```diff
@@ -1337,7 +1337,7 @@ def convert_sliplist(self, node):
         except AttributeError:
             lst = []
         else:
-            lst = [node_to_dict(n)['slip'] for n in slip_list.nodes]
+            lst = [list(node_to_dict(n).values())[0] for n in slip_list.nodes]
```

Comment:
Perhaps it was clearer without this change. I made this in order to be more robust with respect to the retrieval of the tag by name, because I was trying to consider the fully qualified tag instead of the one without the namespace.

Reply:
I've reverted this change